reactor-netty-core/src/main/java/reactor/netty/Metrics.java

```diff
@@ -33,6 +36,7 @@
  */
 public class Metrics {
 	public static final MeterRegistry REGISTRY = io.micrometer.core.instrument.Metrics.globalRegistry;
+	public static final String OBSERVATION_KEY = "micrometer.observation";
```

Comment:
could use the one from the TLAccessor:
```suggestion
	public static final String OBSERVATION_KEY = ObservationThreadLocalAccessor.KEY;
```
reading a static constant field shouldn't lead to class initialization so it should work without `context-propagation` on the classpath

Reply:
@simonbasle The build fails with the below if I don't have dependency

```
> Task :reactor-netty-core:compileJava
error: cannot access ThreadLocalAccessor
  class file for io.micrometer.context.ThreadLocalAccessor not found
  Consult the following stack trace for details.
  com.sun.tools.javac.code.Symbol$CompletionFailure: class file for io.micrometer.context.ThreadLocalAccessor not found
1 error
```

Reply:
(I would defer to @rstoyanchev's expertise here though, I might have been wrong about that)

Reply:
interesting... nevermind then